packages/ra-ui-materialui/src/list/filter/FilterButton.tsx

```diff
@@ -198,7 +198,7 @@ export const FilterButton = (props: FilterButtonProps): JSX.Element => {
                         })}
                     </MenuItem>
                 )}
-                {hasDisplayedFilters && (
+                {hasFilterValues && (
```

Comment:
You have a new linter warning: `'hasDisplayedFilters' is assigned a value but never used`

Reply:
😩